### PR TITLE
Amb 1422 terraform PR builds

### PR DESCRIPTION
**Pull request by almas-liaqat**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/templates/post-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+              make add-endpoint environment=$(WORKSPACE)
+          fi
+
+      fi
+
+    displayName: Add PR-Domain as KeyValue
+    workingDirectory: "$(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/terraform"
+    retryCountOnTaskFailure: 2
+
+  - bash: |
```
Comment (ChewieCB) [inline azure/templates/post-deploy.yml:66]: Can we combine this step with the one above it? Seems like there's a lot of repeated code here

File: azure/templates/post-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+      if ! [[ $APIGEE_ENVIRONMENT =~ .*-*sandbox ]]; then
```
Comment (ChewieCB) [inline azure/templates/post-deploy.yml:68]: Can we combine this step with the one above it? Seems like there's a lot of repeated code here

**Review by ChewieCB — CHANGES_REQUESTED** [review]

File: azure/templates/post-deploy.yml (hunk 2 above)
Comment (almas-liaqat) [reply azure/templates/post-deploy.yml:68]: I agree that code is repeated but both of htem are in separate working directories, so we cant combine them into one

**Review by ChewieCB — DISMISSED** [review]

File: azure/azure-release-pipeline.yml [hunk 3]
```diff
@@ -54,7 +54,7 @@ extends:
           - internal_qa
           - internal_qa_sandbox
         post_deploy:
-          - template: ./templates/run-tests.yml
+          - template: ./templates/post-deploy.yml
       - environment: sandbox
         proxy_path: sandbox
       - environment: int
```
Comment (RyanThomas1214) [inline azure/azure-release-pipeline.yml:60]: Add the post_deploy template to this INT deployment as well

**Review by ChewieCB — APPROVED** [review]

**Pull request merged**